Source/Core/DiscIO/Blob.cpp

```diff
@@ -177,23 +178,24 @@ std::unique_ptr<IBlobReader> CreateBlobReader(const std::string& filename)
   if (cdio_is_cdrom(filename))
     return DriveReader::Create(filename);
 
-  if (!File::Exists(filename))
+  File::IOFile file(filename, "rb");
+  u32 magic;
```

Comment:
I've added a comment here that explains the file format identification in detail. I'd like to keep the word `magic` in the variable name, though.